lib/components/src/blocks/Story.tsx

```diff
@@ -10,14 +9,20 @@ export enum StoryError {
   NO_STORY = 'No component or story to display',
 }
 
+/** error message for Story with null storyFn
+ * if the story id exists, it must be pointing to a non-existing story
+ *  if there is assigned story id, the story must be empty
+ */
+const MISSING_STORY = (id: string) => (id ? `Story "${id}" doesn't exist.` : StoryError.NO_STORY);
+
 interface CommonProps {
   title: string;
   height?: string;
   id: string;
 }
 
 type InlineStoryProps = {
-  storyFn: React.ElementType;
+  storyFn: () => React.ElementType;
```

Comment:
@atanasster Why not just make this `React.ElementType`? It will clean up the typing below too.

Reply:
And you can do everything inline too

Reply:
thanks for the feedback - simplified the code now